server/services/courseService.test.ts

```diff
@@ -61,6 +61,21 @@ describe('CourseService', () => {
     })
   })
 
+  describe('getCourseNames', () => {
+    it('returns a list of all course names', async () => {
+      const courses = courseFactory.buildList(3)
+      const courseNames = courses.map(course => course.name)
+      courseClient.findCourseNames.mockResolvedValue(courseNames)
```

Comment:
This is nice for documentation purposes, but I wonder if it's worth just going straight to an array literal and skipping the first `const` to make the test quicker? Perhaps this is more future proof in case we ever changed the type of `Course['name']`

Reply:
I had a mix initially where I did do that (array of strings) but then also did it this way where there were already courses created using the factory.  Decided to stick with one way for consistency, no other reason than that though so not precious to changing now or in future if it makes a difference.